src/perception/aruco_detect.hpp

```diff
@@ -60,80 +60,76 @@
 #include <opencv2/aruco.hpp>
 #include <opencv2/calib3d.hpp>
 
-#include <list>
 #include <string>
+#include <numeric>
+#include <optional>
+#include <unordered_map>
 #include <boost/algorithm/string.hpp>
 #include <boost/shared_ptr.hpp>
 
-#include "mrover/MarkerInfo.h"
-
 typedef boost::shared_ptr<fiducial_msgs::FiducialArray const> FiducialArrayConstPtr;
```

Comment:
boost shared ptr 👀

Reply:
lol that was left over from the old code, gotta use `std::shared_ptr`